besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -720,6 +720,13 @@ void setBannedNodeIds(final List<String> values) {
       description = "Logging verbosity levels: OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL")
   private final Level logLevel = null;
 
+  @SuppressWarnings({"FieldCanBeFinal"})
+  @Option(
+      names = {"--force-color-enabled"},
```

Comment:
If there's been discussion about this elsewhere I'm happy to be overruled but personally I would have just called this `--color-enabled`.